platform/security/security-migratable/src/main/java/org/codice/ddf/security/migratable/impl/SecurityMigratable.java

```diff
@@ -39,74 +31,83 @@
 /**
  * This class handles the export process for all Security system files
  */
-public class SecurityMigratable extends DescribableBean implements ConfigurationMigratable {
-
+public class SecurityMigratable implements Migratable {
     private static final Logger LOGGER = LoggerFactory.getLogger(SecurityMigratable.class);
 
+    /**
+     * Holds the current export version.
+     * <p>
+     * 1.0 - initial version
+     */
+    private static final String VERSION = "1.0";
+
     private static final Path PDP_POLICIES_DIR = Paths.get("etc", "pdp");
 
-    private static final List<Path> PROPERTIES_FILES = ImmutableList.of(Paths.get("etc",
-            "ws-security",
-            "server",
-            "encryption.properties"),
+    private static final List<Path> PROPERTIES_FILES = ImmutableList.of( //
```

Comment:
❓ I know it wasn't before, but do we want to consider making this configurable through blueprint?

Reply:
I am not sure I am a fan of that. When it comes to supporting upgrade later, it won't be as simple as copying like it is now. Knowledge of the files and what to do will be important. In addition addition of new files to the list would typically increase the version number and now we would be maintaining information in 2 places as opposed to one.

Reply:
Not a fan of doing that. See previous reply